noxfile.py

```diff
@@ -221,6 +222,63 @@ def upload_wheel(session: nox.Session) -> None:
     session.run("twine", "upload", "dist/*", external=True)
 
 
+@nox.session
+def bundle_app(session: nox.Session) -> None:
+    """Bundle a standalone executable."""
+    session.install(
+        "jinja2",
+        "lxml",
+        "pygments",
+        "pyinstaller",
+    )
+    session.install("-e", ".")
+    if not os.path.exists("build"):
+        os.mkdir("build")
```

Comment:
Can be simplified as:

```suggestion
    os.makedirs("build", exists_ok=True)
```